catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/databases/TableResource.java

```diff
@@ -471,6 +482,133 @@ public Table addSampleData(
     return addHref(uriInfo, table);
   }
 
+  @PUT
```

Comment:
@harshach this endpoint does not work on my end 500 error. Am I missing something?

```
curl --location --request PUT 'http://localhost:8585/api/v1/tables/{id}/tableProfilerConfig' \
--header 'Content-Type: application/json' \
--data-raw '{
    "profileSample": 70
}'
```